runtime/gc_base/ObjectAccessBarrier.cpp

```diff
@@ -1345,38 +1345,77 @@ MM_ObjectAccessBarrier::getLockwordAddress(J9VMThread *vmThread, J9Object *objec
  * The new object was just allocated inside the VM, so all fields are NULL.
  * @TODO This does not currently check if the fields that it is reading are volatile.
  */
-void 
+void
 MM_ObjectAccessBarrier::cloneObject(J9VMThread *vmThread, J9Object *srcObject, J9Object *destObject)
 {
+	copyObject(vmThread, J9GC_J9OBJECT_CLAZZ(srcObject), srcObject, sizeof(J9Object), destObject, sizeof(J9Object));
+}
```

Comment:
we have this macro: J9_OBJECT_HEADER_SIZE

Reply:
I see that original code was using sizeof(J9Object), then it should be ok,  too.